syft/pkg/cataloger/golang/licenses_test.go

```diff
@@ -25,6 +25,14 @@ import (
 )
 
 func Test_LicenseSearch(t *testing.T) {
+	sc := &licenses.ScannerConfig{
+		CoverageThreshold: 75,
+		Scanner:           licensecheck.Scan,
+	}
+	scanner, err := licenses.NewScanner(sc)
+	require.NoError(t, err)
+	ctx := licenses.SetContextLicenseScanner(context.Background(), scanner)
```

Comment:
we could probably add a helper function to `pkgtest`

Reply:
Looks like we already had one. I've moved all instances of creating the scanner to consuming this already existing context function